src/cmd/describegpt.rs

```diff
@@ -274,10 +332,22 @@ fn run_inference_options(
             get_tags_prompt(stats_str, frequency_str, args_json)
         };
         messages = get_messages(&prompt, &dictionary_completion_output);
-        println!("Generating tags from OpenAI API...");
+        eprintln!("Generating tags from OpenAI API...");
         completion = get_completion(api_key, &messages, args)?;
         completion_output = get_completion_output(&completion)?;
-        println!("Tags output:\n{completion_output}");
+        eprintln!("Received tags completion output.");
+        process_output("tags", &completion_output, &mut total_json_output, args);
+    }
+
+    if args.flag_json {
+        // Print all JSON output
+        let formatted_output =
+            replace_escape_chars(&serde_json::to_string_pretty(&total_json_output).unwrap());
+        println!("{}", formatted_output);
```

Comment:
@jqnatividad Should `println!` be skipped when a user uses `--output` (also for line 271)?

Reply:
Yes.
Maybe you could do something like this:

https://github.com/jqnatividad/qsv/blob/2fa8ffcd424aaf8f0daacc82eff958120301526d/src/cmd/snappy.rs#L143-L152

And then you just have a generic `output_writer` for creating the output:

```rust
output_writer.write(bytes!("testme\n"));
...
...
output_writer.flush()?;
```